### PR TITLE
Vacms 1020 1021 1057 quick items et

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #1020 #1021 #1057 

## Testing done
Rebuild then visual

## QA steps
**1020**
 - [x] enable module missing message fixer.
 - [x] As an admin, go to `admin/config/system/module-missing-message-fixer`
 - [x] Select all three items, then click `Remove these errors!`
 - Clear cache
 - [x] Go to `admin/reports/status` and visually verify that missing module notices do not appear at top of page
- [x] uninstall module missing message fixer.

**1021**
 - [x] As an admin, disable the views_ui module via the module uninstall ui
 - [x] Clear cache, and go to `admin/reports/status` - visually verify page is not wsod

**1057**
 - [x] As an admin, go to /node/901/edit and scroll to Main content.
 - [x] Visually verify the paragraph titles do not have a second line (group title)

## Post prod deployment steps
**1020**
 - [ ] enable module missing message fixer.
 - [ ] As an admin, go to `admin/config/system/module-missing-message-fixer`
 - [ ] Select all three items, then click `Remove these errors!`
 - Clear cache
 - [ ] Go to `admin/reports/status` and visually verify that missing module notices do not appear at top of page
- [ ] uninstall module missing message fixer.

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -137,6 +138,7 @@ module:
   slick_entityreference: 0
   social_media_links: 0
   social_media_links_field: 0
+  syslog: 0
```
Comment (swirtSJW) [inline config/sync/core.extension.yml]: Were you expecting this?

Comment (ethanteague) [reply config/sync/core.extension.yml]: No. Dunno why that occurred. Lemme take out, @swirtSJW

**Comment by swirtSJW** [comment]
Looks like there is a conflict with composer

**Comment by ethanteague** [comment]
> Looks like there is a conflict with composer

Just pushed the fix, @swirtSJW

File: config/sync/core.extension.yml (hunk 1 above)
Comment (swirtSJW) [inline config/sync/core.extension.yml]:
```suggestion
```

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -97,6 +97,7 @@ module:
   migrate_tools: 0
   migration_tools: 0
   moderation_sidebar: 0
+  module_missing_message_fixer: 0
```
Comment (swirtSJW) [inline config/sync/core.extension.yml]: I'm suggesting to leave this out so it stays disabled, then we just enable it in the ui when we need it and uninstall it ourselves  when we are done.

```suggestion
```

File: config/sync/core.extension.yml (hunk 1 above)
Comment (swirtSJW) [reply config/sync/core.extension.yml]: I removed it

**Comment by swirtSJW** [comment]
Just waiting on tests, followed by QA on dev.

**Comment by swirtSJW** [comment]
Tested out on http://pr1100.ci.cms.va.gov/  all good.   merging now

**Pull request merged**